python/dgl/data/chem/csv_dataset.py

```diff
@@ -3,12 +3,12 @@
 import dgl.backend as F
 import numpy as np
 import os
-import pickle
 import sys
 
-from dgl import DGLGraph
 from .utils import smile_to_bigraph
-
+from ..utils import save_graphs, load_graphs
+from ... import backend as F
+from ...graph import DGLGraph
 
 class CSVDataset(object):
```

Comment:
Minjie suggested to change the class name to MoleculeCSVDataset before. (https://github.com/dmlc/dgl/pull/776#discussion_r315423098)

Reply:
Done.